x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/api_key/50_cross_cluster.yml

```diff
@@ -0,0 +1,343 @@
+---
+setup:
+  - skip:
+      features: headers
+
+  - do:
+      cluster.health:
+        wait_for_status: yellow
+
+  - do:
+      security.put_role:
+        name: "admin_role"
+        body: >
+          {
+            "cluster": ["manage_security"]
+          }
+
+  - do:
+      security.put_user:
+        username: "admin_user"
+        body: >
+          {
+            "password" : "x-pack-test-password",
+            "roles" : [ "admin_role" ],
+            "full_name" : "Admin user"
+          }
+
+---
+teardown:
+  - do:
+      security.delete_role:
+        name: "admin_role"
+        ignore: 404
+
+  - do:
+      security.delete_user:
+        username: "admin_user"
+        ignore: 404
+
+---
+"Test create a cross-cluster API key":
```

Comment:
The YAML tests focus mostly on happy paths since it does not feel necessary to cover all failure scenarios (permission etc) here. The failure scenarios are already covered extensively with unit, integ and REST tests. So I think the YAML tests can focus on the request and response shapes.